blazor/treemap/accessibility.md

```diff
@@ -37,4 +37,4 @@ This attribute provides text label with some default description for the followi
 </tr>
 </table>
 
-Change this default description using the [`Description`](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.TreeMap.SfTreeMap-1.html#Syncfusion_Blazor_TreeMap_SfTreeMap_1_Description) property available in the [TreeMapLegendSettings](https://help.syncfusion.com/cr/aspnetcore-blazor/Syncfusion.Blazor.TreeMap.TreeMapLegendSettings.html), [TreeMapTitleSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.TreeMap.TreeMapTitleSettings.html), [TreeMapSubTitleSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.TreeMap.TreeMapSubtitleSettings.html), and [SfTreeMap](https://help.syncfusion.com/cr/blazor). It helps screen readers to read for assistive purpose.
\ No newline at end of file
+Change this default description using the [`Description`](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.TreeMap.SfTreeMap-1.html#Syncfusion_Blazor_TreeMap_SfTreeMap_1_Description) property available in the [TreeMapLegendSettings](hhttps://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.TreeMap.TreeMapLegendSettings.html), [TreeMapTitleSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.TreeMap.TreeMapTitleSettings.html), [TreeMapSubTitleSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.TreeMap.TreeMapSubtitleSettings.html), and [SfTreeMap](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.TreeMap.SfTreeMap-1.html#Syncfusion_Blazor_TreeMap_SfTreeMap_1__ctor). It helps screen readers to read for assistive purpose.
```

Comment:
Under SfTreeMap use this link - https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.TreeMap.SfTreeMap-1.html

<img width="662" alt="image" src="https://github.com/syncfusion-content/blazor-docs/assets/95272306/a2d41aff-ee66-4638-8617-f583df3ccec1">

Reply:
resolved